applications/NXapm.nxdl.xml

```diff
@@ -319,21 +317,37 @@
                 included when the atom probe session started. If the exact duration of
                 the measurement is not relevant, start_time only should be used.
                 
-                Often though it is useful to specify both start_time and end_time to
-                capture more detailed bookkeeping of the experiment. The user should
-                be aware that even with having both dates specified, it may not be
-                possible to infer how long the experiment took or for how long data
-                were collected.
+                The start_time is required to have provide always at least one
```

Comment:
The start_time is required in order to ensure that at least one point in time is provided for full temporal context to ...

Reply:
Edited accordingly